exec/jdbc/src/main/java/org/apache/drill/jdbc/impl/DrillDatabaseMetaDataImpl.java

```diff
@@ -36,6 +42,7 @@
  */
 class DrillDatabaseMetaDataImpl extends AvaticaDatabaseMetaData
```

Comment:
Also, does Drill allow user to specify the ANSI_QUOTE option in JDBC connection string? If not, that means user does not have a way to change ANSI_QOUTE option if they connect through JDBC?  If so, then lots of BI tool will not benefit from this patch, if they want to use double quotes in their sql statement.